### ISSUE TITLE
Abount 3D attention visualization code

### ISSUE THREAD

**Issue by oliver0922** [IDEA-Research/3D-deformable-attention#9]
Thanks for your nice work!
Do you have any plan to release the 3D deformable attention visualization code?

**Comment by LHY-HongyangLi** [comment]
Yes, we will provide the visualization tool, which is a web demo. Here is a preview of the tool:
![vis_attention_all_flatten](https://github.com/IDEA-Research/3D-deformable-attention/assets/89394032/b0360cba-89e9-4f85-8b65-9a451569869b)

**Comment by oliver0922** [comment]
Nice tools.
When will you realease?
